From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robert Svarinskis <rsvarinskis@google.com>
Date: Mon, 4 Dec 2023 09:12:21 +0000
Subject: [PATCH] --EKS-PATCH-- Fix etcd storage_events_received_total metric

 Fix etcd storage_events_received_total metric not being registered #122168

Cherry-picked from
https://github.com/kubernetes/kubernetes/commit/ceb7ca93875cd74bea60500812cf7f024d3b1f2d

 registered
---
 .../src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go    | 1 +
 1 file changed, 1 insertion(+)

diff --git a/staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go b/staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go
index fadc87d53de..7bf597d3a4b 100644
--- a/staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go
+++ b/staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go
@@ -167,6 +167,7 @@ func Register() {
 		legacyregistry.MustRegister(objectCounts)
 		legacyregistry.MustRegister(dbTotalSize)
 		legacyregistry.CustomMustRegister(storageMonitor)
+		legacyregistry.MustRegister(etcdEventsReceivedCounts)
 		legacyregistry.MustRegister(etcdBookmarkCounts)
 		legacyregistry.MustRegister(etcdLeaseObjectCounts)
 		legacyregistry.MustRegister(listStorageCount)
